tools/codegen/dest/lazy_ir.py

```diff
@@ -230,6 +236,13 @@ class GenLazyNativeFuncDefinition:
     backend_index: BackendIndex
     tensor_class: str
     gen_forced_fallback_code: bool
+    backend_namespace: str = "torch::lazy"
+    get_tensorlist: str = "GetTensorList"
+    get_tensor_or_wrap_number: str = "GetLtcTensorOrCreateForWrappedNumber"
+    try_get_tensor: str = "TryGetLtcTensor"
+    metrics_counter: str = 'TORCH_LAZY_FN_COUNTER("lazy::")'
+    create_tensor: str = "LazyTensor::Create"
+    create_from_first_tensor: bool = False
```

Comment:
I think this is ok as a short-term crutch (and bundling them all here with their defaults makes it easy to know what to kill later). Do you think these will all go away in the next few months? Or do you think some of them will be permanent differences between XLA / TS (for stuff like metrics counter, maybe? idk).

And quick related question - how does XLA actually plumb this info into the codegen? (do these values go in the yaml?)

Reply:
I think all of this is intended to go away after the migration completes, but possibly some of it would stay.

The plumbing part is part of what I was wanting to clean up.  By default i could plumb these all individually as arguments to the run_gen_lazy_tensor function, but it will be a lot of arguments.

Reply:
I think most (if not all) of these will go away after we migrate to the `LazyTensor` class in a couple months.